posthog/settings/__init__.py

```diff
@@ -113,3 +113,5 @@
 
 # Lastly, cloud settings override and modify all
 from posthog.settings.cloud import *  # noqa: F401
+
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
https://github.com/PostHog/posthog/pull/10369 already did this

Note we shouldn't be setting things in this file - data_stores.py is for data store-related configuration